pkg/operators/kubemanager/kubemanager.go

```diff
@@ -337,6 +343,171 @@ func (m *KubeManagerInstance) EnrichEvent(ev any) error {
 	return nil
 }
 
+func (k *KubeManager) GlobalParams() api.Params {
+	return apihelpers.ParamDescsToParams(k.GlobalParamDescs())
+}
+
+func (k *KubeManager) InstanceParams() api.Params {
+	return apihelpers.ParamDescsToParams(k.ParamDescs())
+}
+
+func (k *KubeManager) InstantiateDataOperator(gadgetCtx operators.GadgetContext, paramValues api.ParamValues) (
+	operators.DataOperatorInstance, error,
+) {
+	params := k.ParamDescs().ToParams()
+	err := params.CopyFromMap(paramValues, "")
+	if err != nil {
+		return nil, err
+	}
+
+	traceInstance := &KubeManagerInstance{
+		manager:            k,
+		enrichEvents:       false,
+		attachedContainers: make(map[string]*containercollection.Container),
+		params:             params,
+		gadgetCtx:          gadgetCtx,
+		id:                 uuid.New().String(),
+
+		eventWrappers: make(map[datasource.DataSource]*compat.EventWrapperBase),
+	}
+
+	// hack - this makes it possible to use the Attacher interface
+	var ok bool
+	traceInstance.gadgetInstance, ok = gadgetCtx.GetVar("ebpfInstance")
+	if !ok {
+		return nil, fmt.Errorf("getting ebpfInstance")
+	}
+
+	activate := false
+
+	// Check, whether the gadget requested a map from us
+	if t, ok := gadgetCtx.GetVar(gadgets.MntNsFilterMapName); ok {
+		if _, ok := t.(*ebpf.Map); ok {
+			gadgetCtx.Logger().Debugf("gadget requested map %s", gadgets.MntNsFilterMapName)
+			activate = true
+		}
+	}
+
+	// Check for override - currently needed for tchandlers
```

Comment:
AFAIU, tc programs could run per container - so they need the container subscriptions / attacher interface, even if they don't use the usual maps.

Reply:
Aaaahh ok, ok. Got it. I think the `overrideEnricher` name is clear to me. Something like `NeedContainerEvents` or subscribe, etc., would be more clear.

Reply:
You're right, I'll change it.

Reply:
> I am not sure of the semantic of this variable overrideEnricher... could you add a comment next to its definition?

Agree that it feels a bit weird - I hope we can optimize that later (I think it might be a good idea to create a DataSource from the container subscriptions, for example).